json-guides/bulkhead.json

```diff
@@ -165,14 +165,66 @@
             ]
         },
         {
-            "name": "AddingAsyncAnnotation",
-            "title": "Adding the @Asynchronous annotation",
+            "name": "AsyncBulkheadAnnotation",
+            "title": "Adding the @Asynchronous annotation with Bulkhead",
             "description": [  
-                "Thread pool style Bulkhead"        
+                "When <code>@Bulkhead</code> is used with <code>@Asynchronous</code>, the thread pool isolation approach will be used. The thread pool approach allows to configure the maximum concurrent requests together with the waiting queue size. When a request cannot be added to the waiting queue, BulkheadException will be thrown.</br>",
```

Comment:
You need to begin with transition from the previous step explaining what you're changing/fixing/adding.